rest/backend/src/test/scala/io/udash/rest/server/EndpointsIntegrationTest.scala

```diff
@@ -134,11 +134,11 @@ class EndpointsIntegrationTest extends UdashSharedTest with BeforeAndAfterAll wi
     "handle connection refused" in {
       val eventualResponse =
         inexistentServerConnector.send("/non/existing/path", RESTConnector.HttpMethod.POST, Map.empty, Map.empty, null)
-      intercept[ConnectException](await(eventualResponse))
+      intercept[ConnectException](Await.result(eventualResponse, 10 seconds))
 
       val eventualResponse2 =
         inexistentServerConnector.send("/non/existing/path", RESTConnector.HttpMethod.POST, Map.empty, Map.empty, "lol")
-      intercept[ConnectException](await(eventualResponse2))
+      intercept[ConnectException](Await.result(eventualResponse2, 10 seconds))
```

Comment:
We can use a global (or even scaled) implicit timeout with the `ScalaFutures` syntax here:
```scala
eventualResponse2.failed.futureValue shouldBe a [ConnectException]
```

Reply:
I don't want the global timeout, in this test I want to use this specific value.

Reply:
You can introduce an implicit in the local scope. This specific value occurs at least twice here ;)